Fun.Blazor/DslWatch.fs

```diff
@@ -18,25 +33,163 @@ type html with
             ]
             ,[])
 
-
+    /// <summary>
+    /// Renders Nodes dynamically based on what is emitted by the store
+    /// </summary>
+    /// <example>
+    /// <code lang="fsharp">
+    /// let View (hook: IComponentHook) =
+    ///   let store = hook.UseStore 10
+    /// 
+    ///   Observable.interval (TimeSpan.FromSeconds(2.))
+    ///   // update the store every two seconds
+    ///   |> Observable.subscribe store.Publish
+    /// 
+    ///   let _view num =
+    ///     p() { childContent $"Number: {num}" }
+    /// 
+    ///   html.watch(store, _view)
+    /// </code>
+    /// </example>
     static member watch (store: IStore<'T>, render: 'T -> Node) = html.watch (store.Observable, render, store.Current)
+    /// <summary>
+    /// Renders Nodes dynamically based on what is emitted by the store
+    /// </summary>
+    /// <example>
+    /// <code lang="fsharp">
+    /// let View (hook: IComponentHook) =
+    ///   let store = hook.UseStore 10
+    /// 
+    ///   Observable.interval (TimeSpan.FromSeconds(2.))
+    ///   // update the store every two seconds
+    ///   |> Observable.subscribe store.Publish
+    /// 
```

Comment:
We should add 
|> hook.AddDispose
Or use Observable.add so it will return unit here.

Reply:
fixed it